app/main.py

```
@@ -135,16 +169,22 @@ async def snapshot_builder(base_url: str) -> dict[str, object]:
                 sync_service.stop()
+                if daily_musicatlas_playlist_task is not None:
+                    await asyncio.wait_for(daily_musicatlas_playlist_task, timeout=5)
                 if sync_task is not None:
                     await asyncio.wait_for(sync_task, timeout=5)
             except asyncio.TimeoutError:
+                if daily_musicatlas_playlist_task is not None:
+                    daily_musicatlas_playlist_task.cancel()
                 if sync_task is not None:
                     sync_task.cancel()
             except Exception:
                 pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)